### PR TITLE
[AzureLoadTesting] Updated the package, README and Tests

### PR DESCRIPTION
**Pull request by venkatr21**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/loadtestservice/Azure.ResourceManager.LoadTesting/README.md [hunk 1]
```diff
@@ -30,6 +30,16 @@ dotnet add package Azure.ResourceManager.LoadTesting --prerelease
...
+```C#
+using Azure.Core;
+using Azure.Identity;
+using Azure.ResourceManager;
+
+// Code omitted for brevity
+
+ArmClient armClient = new ArmClient(new DefaultAzureCredential());
+```
+
```
Comment (Yao725) [inline sdk/loadtestservice/Azure.ResourceManager.LoadTesting/README.md:42]: We already added the link for authentication, so I think we could remove this sample code.

File: sdk/loadtestservice/Azure.ResourceManager.LoadTesting/README.md [hunk 2]
```diff
@@ -44,6 +54,121 @@ Documentation is available to help you learn how to use this package:
 
 ## Examples
 
+### Create a new Azure Load Testing resource
```
Comment (Yao725) [inline sdk/loadtestservice/Azure.ResourceManager.LoadTesting/README.md:57]: I want to share the process about how we add samples (use samples in Cdn as example).

- Add a [cs](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cdn/Azure.ResourceManager.Cdn/tests/Samples/Sample1_ManagingCdnOriginGroups.cs) file under tests/Samples to store the sample code
- Add the [tests](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cdn/Azure.ResourceManager.Cdn/tests/Samples/Sample1_ManagingCdnOriginGroups.cs#L20-L58) to show how to CRUD the sample resource and mark the tests as ignored.
- Use [#region](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cdn/Azure.ResourceManager.Cdn/tests/Samples/Sample1_ManagingCdnOriginGroups.cs#L25) and [#endregion](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cdn/Azure.ResourceManager.Cdn/tests/Samples/Sample1_ManagingCdnOriginGroups.cs#L57) to organize the sample code block and put the same [region name](https://github.com/Azure/azure-sdk-for-net/blame/main/sdk/cdn/Azure.ResourceManager.Cdn/samples/Sample1_ManagingCdnOriginGroups.md#L38) in the README file
- run `eng\scripts\Update-Snippets.ps1 loadtestservice` to update the samples in README

Could we please also follow the same process here? It ensures that our sample code could compile successfully and is up to date.

File: sdk/loadtestservice/Azure.ResourceManager.LoadTesting/README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
-# Microsoft Azure Load Testing management client library for .NET
```
[the remark this answers is not in the document]
Comment (ninallam) [reply sdk/loadtestservice/Azure.ResourceManager.LoadTesting/README.md:1]: @Yao725 We typically don't use "Microsoft" brand when we are in the context of Microsoft already (learn.microsoft.com in this case). All other SDKs and public docs also just mentions Azure Load Testing only. Please let us know if you see any concern.

File: sdk/loadtestservice/Azure.ResourceManager.LoadTesting/tests/Samples/Sample2_LoadTestingQuotaOperations.cs [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+            // Use the quotaCollection for all the quota operations.
+            #endregion Snippet:LoadTesting_GetQuotaCollection
+        }
+
+        [Test]
+        [Ignore("Only verifying that the sample builds")]
+        public async Task GetLoadTestingQuotaBucket()
+        {
+            #region Snippet:LoadTesting_GetQuotaBucket
+            LoadTestingQuotaCollection QuotaCollection = _subscription.GetAllLoadTestingQuota(AzureLocation.WestUS2);
```
Comment (Yao725) [inline sdk/loadtestservice/Azure.ResourceManager.LoadTesting/tests/Samples/Sample2_LoadTestingQuotaOperations.cs:35]: Same as above

File: sdk/loadtestservice/Azure.ResourceManager.LoadTesting/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Yao725) [reply sdk/loadtestservice/Azure.ResourceManager.LoadTesting/README.md:1]: Need @ArthurMa1978's confirmation about this change.

File: sdk/loadtestservice/Azure.ResourceManager.LoadTesting/tests/Samples/Sample2_LoadTestingQuotaOperations.cs [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+            Console.WriteLine(checkAvailabilityResult.Value.IsAvailable);
+            #endregion Snippet:LoadTesting_CheckQuotaAvailability
+        }
+
+        [SetUp]
+        [Ignore("Only verifying that the sample builds")]
+        protected async Task initialize()
+        {
+            ArmClient armClient = new ArmClient(new DefaultAzureCredential());
+            SubscriptionResource _subscription = await armClient.GetDefaultSubscriptionAsync();
```
[the remark this answers is not in the document]
Comment (venkatr21) [reply sdk/loadtestservice/Azure.ResourceManager.LoadTesting/tests/Samples/Sample2_LoadTestingQuotaOperations.cs]: Have updated this typo

File: sdk/loadtestservice/Azure.ResourceManager.LoadTesting/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (ArthurMa1978) [reply sdk/loadtestservice/Azure.ResourceManager.LoadTesting/README.md:1]: This is a suggestion from [Acrolinx](https://review.learn.microsoft.com/en-us/help/contribute/contribute-get-started-setup-acrolinx-vscode?branch=main), all readme follow the same pattern.

[the remark this answers is not in the document]
Comment (venkatr21) [reply sdk/loadtestservice/Azure.ResourceManager.LoadTesting/README.md]: @ArthurMa1978 @Yao725 I have updated the readme as per the review.

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**